csv/ParameterDefs.csv

```diff
@@ -3147,15 +3147,15 @@ NUTNR_M_GLIDER,Gliders,sci_suna_nitrate_um,PD3704,quantity,,float32,,µmol/L,,Ni
-FDCHP_A_DCL,FDCHP,fdchp_a_fluxmom_alongwind,PD3721,function,,float32,,m2 s-2,-9999999,Along-Wind Component of Momentum Flux,4,,PFID195,"{""timestamp"":""PD7"",""sonicU"":""PD3471"",""sonicV"":""PD3472"",""sonicW"":""PD3473"",""heading"":""PD3480"",""rateX"":""PD3321"",""rateY"":""PD3322"",""rateZ"":""PD3323"",""accX"":""PD3475"",""accY"":""PD3476"",""accZ"":""PD3477"",""lat"":""CC_latitude""}",,,FLUXMOM-U_L2,"FLUXMOM-U_L2, the along-wind component of the momentum flux, from the FDCHP instrument, which collects 20 minutes of data every hour. There is one FLUXMOM-U_L2 value calculated for each 20 minute dataset of 12000 data records.",,,L2
-FDCHP_A_DCL,FDCHP,fdchp_a_fluxmom_crosswind,PD3722,function,,float32,,m2 s-2,-9999999,Cross-Wind Component of Momentum Flux,4,,PFID196,"{""timestamp"":""PD7"",""sonicU"":""PD3471"",""sonicV"":""PD3472"",""sonicW"":""PD3473"",""heading"":""PD3480"",""rateX"":""PD3321"",""rateY"":""PD3322"",""rateZ"":""PD3323"",""accX"":""PD3475"",""accY"":""PD3476"",""accZ"":""PD3477"",""lat"":""CC_latitude""}",,,FLUXMOM-V_L2,"FLUXMOM-V_L2, the cross-wind component of the momentum flux, from the FDCHP instrument, which collects 20 minutes of data every hour. There is one FLUXMOM-V_L2 value calculated for each 20 minute dataset of 12000 data records.",,,L2
-FDCHP_A_DCL,FDCHP,fdchp_a_time_L1,PD3723,function,,float32,,s,-9999999,L1 Timestamps,4,,PFID197,"{""timestamp"":""PD7""}",,,TIME_L1-AUX,"The time metadata product TIME_L1-AUX associated with the L1 wind and temperature data products from the FDCHP instrument, which collects 20 minutes of data every hour. For each of the L1 data products, 30 seconds of data is stripped out from the beginning and end of each 20 minute, 12000 record dataset. The TIME_L1-AUX values are the remaining timestamps for each dataset.",,,
-FDCHP_A_DCL,FDCHP,fdchp_a_time_L2,PD3724,function,,float32,,s,-9999999,L2 Timestamps,4,,PFID198,"{""timestamp"":""PD7""}",,,TIME_L2-AUX,"The time metadata product TIME_L2-AUX associated with the L2 flux data products from the FDCHP instrument. FDCHP collects 20 minutes of data every hour; for each of the L2 flux data products, one data value is calculated for each 20 minute dataset. The TIME_L2-AUX values are the median timestamps for each dataset.",,,
+FDCHP_A_DCL,FDCHP,fdchp_a_tmpatur,PD3716,function,"[""-obs"",""fdchp_a_time_L1""]",float32,,ºC,-9999999,Sonic Air Temperature,4,,PFID190,"{""timestamp"":""PD7"",""sonicT"":""PD3474""}",,,TMPATUR_L1,Sonic Air Temperature refers to air temperature associated with eddies that flow over the ocean surface and is used to determine vertical fluxes of heat associated with the eddies.,Sonic Air Temperature,Science Data,L1
```

Comment:
How does switching from PD1053(air_temperature) to PD3474(fdchp_speed_of_sound_sonic) to satisfy PD3716(fdchp_a_tmpatur) help?

Reply:
For history, see comments 19 and 23 in the 13402 redmine ticket. Short story is, PD1053 is the wrong input to use for this calculation. PD3474 is the correct input. So changing it makes the fdchp_a_tmpatur results correct and prevents errors from erroneously trying to resolve a stream which contains PD1053.